packages/custom-functions-metadata/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "plugins": [
+    "office-addins"
+  ],
+  "extends": [
+    "plugin:office-addins/recommended"
+  ]
+}
```

Comment:
minor: add a newline since usually you want the last line to be empty. The diff shows a newline is missing at the end of this line.